Engine_Revit_UI/Query/Material.cs

```diff
@@ -38,20 +37,21 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        static public oM.Environment.Materials.Material Material(this CompoundStructureLayer compoundStructureLayer, Document document, BuiltInCategory builtInCategory = Autodesk.Revit.DB.BuiltInCategory.INVALID, PullSettings pullSettings = null)
+        static public oM.Physical.Properties.Construction.Layer Material(this CompoundStructureLayer compoundStructureLayer, Document document, BuiltInCategory builtInCategory = Autodesk.Revit.DB.BuiltInCategory.INVALID, PullSettings pullSettings = null)
```

Comment:
This shall be renamed from Material to Layer and moved to Engine_Revit_UI/Query/Layer.cs